src/stores/TransactionStore.js

```diff
@@ -61,27 +77,19 @@ export default class TransactionStore {
   }) => {
     try {
       // Construct params
-      const { nbotOwner, exchangeRate } = this.app.wallet;
-      if (!nbotOwner) {
-        throw new Error('exchanger not existed');
-      }
-      if (!exchangeRate) {
-        throw new Error('excahngeRate not existed');
-      }
       const paramsHex = web3EthAbi.encodeParameters(
         CREATE_EVENT_FUNC_TYPES,
         eventParams,
       ).substr(2);
-
       const data = `0x${CREATE_EVENT_FUNC_SIG}${paramsHex}`;
+
       // Send tx
+      const pbtParams = this.getPayByTokenParams();
       const txid = await promisify(
         nbotMethods.transfer['address,uint256,bytes'].sendTransaction,
         [eventFactoryAddr, escrowAmt, data, {
-          token: getContracts().NakaBodhiToken.address,
-          exchanger: nbotOwner,
-          exchangeRate,
           gas,
+          ...pbtParams,
```

Comment:
`pbtParams` is an object, don't think we can use spread syntax on object?

Reply:
yup it works on objects as well